common/pinot/pinotClient.go

```diff
@@ -172,6 +182,9 @@ func (c *PinotClient) convertSearchResultToVisibilityRecord(hit []interface{}, c
 func (c *PinotClient) getInternalListWorkflowExecutionsResponse(
 	resp *pinot.BrokerResponse,
 	isRecordValid func(rec *p.InternalVisibilityWorkflowExecutionInfo) bool,
+	token *elasticsearch.ElasticVisibilityPageToken,
```

Comment:
We should create a new page token for Pinot or change it to a more generic one. It will be confused if you use the es token in Pinot. And one more question, does this need to be implemented in monorepo?

Reply:
yes we also need to implement this in the mono repo, but those will be similar

Reply:
Pinot token added